__tests__/workers/cdc.ts

```diff
@@ -1126,3 +1129,35 @@ describe('submission', () => {
     ).toEqual([after]);
   });
 });
+
+describe('notification', () => {
+  type ObjectType = Notification;
+  const id = randomUUID();
+  const base: ChangeObject<ObjectType> = {
+    id,
+    userId: '1',
+    type: 'community_picks_granted',
+    title: 'hello',
+    targetUrl: 'target',
+    icon: 'icon',
+    public: true,
+    createdAt: Date.now(),
+  };
+
+  it('should notify new notification', async () => {
```

Comment:
Perhaps we can also add a case where we update the readAt and it should not trigger, just for future proofing.

Reply:
This requires debezium in the test environment and we currently don't support it